pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -217,27 +229,6 @@ public void failed(Throwable throwable) {
         }
     }
 
-    @Override
-    public CompletableFuture<List<String>> getListInBundleAsync(String namespace, String bundleRange) {
```

Comment:
it looks like that we are changing the behaviour of this method.
two questions:
- will be the new admin client (2.7.0) compatible with 2.6.x ?
- was this method working before this change ?

Reply:
I think this will not break the compatibility since this will not change the interface.